content/enterprise/install/automated/active-active.mdx

```diff
@@ -265,6 +263,14 @@ As you scale Terraform Enterprise beyond 2 nodes, you may be adding additional s
 - [Google Cloud - Network Quotas and Limits](https://cloud.google.com/vpc/docs/quota)
 - [Google Cloud - API rate limits](https://cloud.google.com/compute/docs/api-rate-limits)
 
+Depending on your infrastructure and Terraform Enterprise configuration, you may need to configure your application gateway or load balancer for "sticky sessions." For example, an Active/Active deployment on Azure with SAML authentication requires sticky sessions be enabled to ensure the authentication with the SAML server is successful. The terminology for this varies across clouds. Please reference the documentation for your infrastructure.
```

Comment:
My only comment here is: Could we provide the briefest definition of "sticky session" for folks? Or are we assuming that everyone will be familiar with this already? Just checking!

Reply:
Okay, added another sentence w/a definition.